app/controllers/issues_controller.rb

```diff
@@ -105,6 +107,10 @@ def update_status
 
     if Issue::AASM_EVENTS_HUMANIZED.include?(event_str) && (update_event = @issue.aasm.events(permitted: true).find { |event| event.name.to_s.humanize == event_str })
       @issue.public_send("#{update_event.name}!")
+
+      # Send mail to watchers
+      @issue.inform_status_change
```

Comment:
use after callback of AASM for sending emails.

Reply:
done